paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -950,6 +950,74 @@ class TransposeMKLDNNHandler : public MKLDNNHandler {
   std::vector<int> logical_axis_;
 };
 
+template <typename T>
+class AXPYMKLDNNHandler : public platform::MKLDNNHandlerT<T, dnnl::reorder> {
+ public:
+  AXPYMKLDNNHandler(const MKLDNNDeviceContext& dev_ctx,
+                    const mkldnn::engine mkldnn_engine,
+                    platform::Place cpu_place, int n, float alpha)
+      : platform::MKLDNNHandlerT<T, dnnl::reorder>(
+            dev_ctx, mkldnn_engine, cpu_place,
+            platform::CreateKey(dev_ctx, static_cast<int64_t>(n),
```

Comment:
Why this int n has to be cast to int64_t as below if it defined an int ?
     platform::CreateKey(dev_ctx, static_cast<int64_t>(n),
                                platform::MKLDNNGetDataType<T>(), alpha,
                                "-axpy")

Reply:
I have removed this handler as it's not used.